src/js/components/Button/stories/Secondary.js

```diff
@@ -0,0 +1,255 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { User } from 'grommet-icons';
+import { Box, Button, grommet, Grommet, Text } from 'grommet';
+import { deepMerge } from 'grommet/utils';
+import { hpe } from 'grommet-theme-hpe';
+
+const myHpe = deepMerge(hpe, {
+  global: {
+    colors: {
+      green: {
+        light: '#17EBA0',
+        dark: '#008567',
+      },
+      'background-contrast': {
+        dark: '#FFFFFF14',
+        light: '#0000000A',
+      },
+      control: 'green',
+    },
+  },
+  button: {
+    color: 'text-strong',
+    active: {
+      border: {
+        color: 'transparent',
+      },
+      background: {
+        color: 'background-contrast',
+      },
+      color: 'text',
+    },
+    disabled: {
+      background: 'transparent',
+      border: {
+        color: '#DDD',
+      },
+      color: '#DDD',
+      opacity: 1.0,
+    },
+    hover: {
+      default: {
+        background: {
+          color: 'background-contrast',
+        },
+        border: {
+          color: 'background-contrast',
+        },
+      },
+    },
+    default: {
+      color: 'text',
+    },
+    secondary: {
+      border: {
+        color: 'green',
+      },
+    },
+    primary: {
+      background: {
+        color: 'green',
+      },
+    },
+  },
+});
+
+const Secondary = () => (
```

Comment:
This example is a little overwhelming. 
Can we make it leaner and remove the usage of the hpe theme? I think the hpe theme will be confusing for both hpe and grommet users, we can just show how to use the new props merged with grommet theme. 

I'm also not inclining to show multiple usages of Grommet component in the same file, it might cause confusion on how the component should be used. In case we do want to demonstrate a comparison between different themes ThemeContext might be an option.